client/my-sites/themes/install-theme-button.jsx

```diff
@@ -24,7 +25,9 @@ function getInstallThemeUrl( state, siteId ) {
 
 	const atomicSite = isAtomicSite( state, siteId );
 	const siteCanInstallThemes = siteHasFeature( state, siteId, FEATURE_INSTALL_THEMES );
-	if ( atomicSite && siteCanInstallThemes ) {
+	const siteHasFeaturesLoaded = !! getSiteFeatures( state, siteId );
+
+	if ( atomicSite && ( siteCanInstallThemes || ! siteHasFeaturesLoaded ) ) {
```

Comment:
Should we just disable the button if the features have not been loaded yet? Just in case there is an edge scenario in which `siteCanInstallThemes ` ends up being false, so we shouldn't redirect to WP Admin.